src/types.ts

```diff
@@ -457,6 +457,7 @@ export interface OldPaymailAddress {
   created_at: Date;
   updated_at?: Date;
   deleted_at?: Date;
+  address: string;
```

Comment:
Maybe, put it right after the alias & domain.

Reply:
moved